Cargo.toml

```diff
@@ -245,12 +245,13 @@ codegen-units = 1
 inherits = "dev"
```

Comment:
nit: this should probably be called `dev-without-assertions` if it's a variant of the `dev` profile